src/scopeHandler.js

```diff
@@ -8,7 +8,7 @@ import { throwTantrum } from './utilities';
  */
 const alwaysThrowHandler = new Proxy(freeze({}), {
   get(target, prop) {
-    throwTantrum(`unexpected scope handler trap called: ${prop}`);
+    throwTantrum(`unexpected scope handler trap called: ${String(prop)}`);
```

Comment:
Good catch!

Reply:
It's Typescript language server warned me about this.